src/components/CreateWalletForm/index.tsx

```diff
@@ -83,6 +84,7 @@ export class WalletForm extends React.Component<WalletFormProps> {
   private handleSubmit: React.MouseEventHandler<any> = e => {
     this.validateForm();
     if (this.props.wallet.isValid) {
+      this.isSubmitting = true;
```

Comment:
will it be released back to `false`? and where

Reply:
On every init it set to false (Line 18), probably the only case where we should release it back is exception on API call, i believe we should handle exceptions on every API call but at another ticket